tests/unit/components/UI/Text/TimeInput.spec.js

```diff
@@ -5,11 +5,9 @@ describe("TimeInput.vue", () => {
   let wrapper = mount(TimeInput);
   it("should render with default values", () => {
     expect(wrapper).toBeTruthy();
-    if (wrapper.vm.localConfig.showHour)
-      expect(
-        wrapper.find('[data-test="hour"]').find('[data-test="input"]').element
-          .value
-      ).toBe("0");
+    expect(wrapper.find('[data-test="hour"]').exists()).toBe(
+      wrapper.vm.localConfig.showHour
+    );
```

Comment:
I don't think this is a valid check - what if showHour was being set incorrectly and hence, the div will incorrectly not be shown? We won't be able to catch it with this test, no?
Ideally, there should be two tests - for both when the hour one is shown and when it is not - and the equality should be checked with `true` or `false` directly.